src/send/SendConfirmation.tsx

```diff
@@ -233,7 +269,10 @@ function SendConfirmation(props: Props) {
     )
   }
 
-  const allowComment = isStablecoin(tokenInfo)
+  const allowComment = newSendScreen
```

Comment:
This would end up allowing comments for any celo token right? I think only celo stables support comments, so this line can just stay as is (we should rename the fn though)

Reply:
The [linear issue](https://linear.app/valora/issue/ACT-944/create-new-send-confirmation-screen) says it should appear if the selected token is on the Celo network, but maybe they meant to keep the current behavior but only for Celo network. I can change it to do that.